src/components/ThreadEnvelope.vue

```diff
@@ -115,6 +115,16 @@
 								:size="20" />
 						</template>
 					</ButtonVue>
+					<ButtonVue v-if="showArchiveButton"
+						:close-after-click="true"
+						type="tertiary-no-background"
+						@click.prevent="onArchive">
+						<template #icon>
+							<ArchiveIcon
+								:title="t('mail', 'Archive message')"
```

Comment:
we dont use the :title anymore, if this is a label, add it below as {{ t('mail', 'Archive message') }}

Reply:
Since this is part of the `ThreadEnvelope` it is for the mouseover event, I just copied it from the `DeleteIcon` a few lines below this